app/jobs/trs/retrieve_trn_job.rb

```diff
@@ -49,6 +49,7 @@ def process_trn_request
       if trn
         trainee.trn_received!(trn)
         trn_request.received!
+        UpdateProfessionalStatusJob.set(wait: 1.minute).perform_later(trainee)
```

Comment:
this ensures the trainees' status (in this case it will be `underAssessment` or `inTraining`) is  properly reported to TRS - it does not go through as part of the TRN request.